src/Tools/onnx-mlir-reduce/CMakeLists.txt

```diff
@@ -7,6 +7,8 @@ add_onnx_mlir_executable(onnx-mlir-reduce
 
   LINK_LIBS PRIVATE
   ${OMLibs}
+  Accelerator
```

Comment:
Can you make sure this lib is excluded from `OMLibs`? I think its cmake currently adds it to the list

Reply:
Done.